### DISCUSSION TITLE
How can we destroy autocomplete once initialized?

### DISCUSSION THREAD

**Discussion by MuhammadTahseenurRehman** [TarekRaafat/autoComplete.js#357] [Q&A]
I am inserting selected city areas in autocomplete but when user changed the city then I am facing a problem that I cannot remove the areas of previously selected city and it is creating problem for me. is there any way that I can destroy autocomplete when user changed its city and then I can put areas of the selected city again?

I shall be very thankful to all who will try to resolve my issue.

**Comment by REDAL** [comment]
@MuhammadTahseenurRehman 
do you mean, when the user select a city, the suggestions list does not disappear ?
